tests/phpunit/integration/Core/Dashboard_Sharing/Active_ConsumersTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Active_ConsumersTest
+ *
+ * @package   Google\Site_Kit\Tests\Core\Dashboard_Sharing
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Dashboard_Sharing;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Dashboard_Sharing\Activity_Metrics\Active_Consumers;
+use Google\Site_Kit\Core\Storage\User_Options;
+use Google\Site_Kit\Tests\TestCase;
+
+class Active_ConsumersTest extends TestCase {
+
+	/**
+	 * @var User_Options
+	 */
+	protected $user_options;
+
+	public function set_up() {
+		parent::set_up();
+		$user_id            = $this->factory()->user->create();
+		$context            = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE );
+		$this->user_options = new User_Options( $context, $user_id );
+		$meta_key           = $this->user_options->get_meta_key( Active_Consumers::OPTION );
+		unregister_meta_key( 'user', $meta_key );
+		// Needed to unregister the instance registered during plugin bootstrap.
+		remove_all_filters( "sanitize_user_meta_{$meta_key}" );
+	}
+
+	public function test_sanitize_callback() {
+		$active_consumers = new Active_Consumers( $this->user_options );
+		$active_consumers->register();
+
+		$this->assertFalse( $this->get_value() );
+
+		// Setting the value to a non-array will result in an empty array.
+		$this->set_value( false );
+
+		$this->assertEquals( array(), $this->get_value() );
```

Comment:
Let's remove an extra empty line between setting and checking values to make it more readable:
```suggestion
		// Setting the value to a non-array will result in an empty array.
		$this->set_value( false );
		$this->assertEquals( array(), $this->get_value() );
```